src/main/java/no/nav/tag/kontakt/oss/geografi/Geografi.java

```diff
@@ -0,0 +1,94 @@
+package no.nav.tag.kontakt.oss.geografi;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+@EqualsAndHashCode
+@Getter
+public class Geografi {
+    private Map<String, List<KommuneEllerBydel>> geografiMap;
+
+    public Geografi(List<NorgGeografi> norgGeografi) {
+        Map<String, List<KommuneEllerBydel>> geografiMap = new HashMap<>();
+        hentFylkesnumre(norgGeografi).forEach(fylkesnummer ->
+                geografiMap.put(fylkesnummer, hentKommunerOgBydeler(norgGeografi, fylkesnummer))
+        );
+        this.geografiMap = geografiMap;
+    }
+
+    private List<String> hentFylkesnumre(List<NorgGeografi> norgGeografi) {
+        return norgGeografi.stream()
+                .filter(this::harIkkeNull)
+                .map(NorgGeografi::getNavn)
+                .filter(this::erFylkesnr)
+                .collect(Collectors.toList());
+    }
+
+    private List<KommuneEllerBydel> hentKommunerOgBydeler(List<NorgGeografi> norgGeografiListe, String fylkesnummer) {
```

Comment:
Kanskje lettere å lese hvis fylkesnummer er første parameter.
"hentKommunerOgBydeler for dette fylkesnummer"